dluhc-component-library/src/components/siteSelectionForm/components/FormPage.tsx

```diff
@@ -29,12 +33,21 @@ const FormPage = ({
         >
           Back
         </button>
-        <button
-          className="bg-green-700 hover:bg-green-800 text-white py-1 px-2"
-          onClick={onContinueClicked}
-        >
-          Save and continue
-        </button>
+        {isLastQuestion ? (
```

Comment:
`Non-blocker` Since we want the styling and structure to be the same it could be worth using `isLastQuestion` in a ternary for the `onClick` and the text rather than the whole html.